vitest.config.js

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig
+({
+    test: {
+        coverage: {
+            provider: 'v8'
+        },
+        setupFiles: './tests/setup.js',
+    },
+});
```

Comment:
Turn on globals so you don't have to import `vi`, `test`, `beforeEach`, etc in every file. Coverage will use v8 by default, so it can be skipped here.
```suggestion
import { defineConfig } from 'vitest/config';

export default defineConfig({
  test: {
    globals: true,
    setupFiles: './tests/setup.js'
  }
});
```